src/app/post-page/post-page.component.html

```diff
@@ -7,36 +7,105 @@
             Next<mat-icon>navigate_next</mat-icon>
         </button>
     </div>
-    <div class="content">
-        <div class="post"> 
-            {{post.id}}
-            <br/>
-            {{post.time_created}}
-            <br/>
+    <mat-card class="content">
+        <mat-card-header>
+            <mat-card-title>{{'#' + post.id}}</mat-card-title>
+            <mat-card-subtitle>{{post.time_created | date:'dd LLLL yyyy, h:mm a'}}</mat-card-subtitle>
+        </mat-card-header>
+        <mat-card-content style="padding:0px 40px;">
             {{post.text}}
-            <br/>
-            {{post.likes}}
-            <br/>
-            <button mat-button (click)="updatePostLikes()">Like</button>
-        </div>
-        <div class="comments">
-            <mat-form-field class="new-comment">
-                <textarea #comment matInput placeholder="Write a comment..."></textarea>
-            </mat-form-field>
-            <button mat-button type="submit" (click)="createNewComment(comment.value)">Comment</button>
-            <div class="old-comments">
-                <div *ngFor="let comment of comments">
-                    {{comment.poster}}
+        </mat-card-content>
+        <mat-card-header>
+            <mat-card-subtitle>
+                {{post.likes === 1 ? post.likes + " Like" : post.likes + " Likes"}}
+                &middot;
+                {{comments.length === 1 ? comments.length + " Comment" : comments.length + " Comments"}}
+            </mat-card-subtitle>
+        </mat-card-header>
+        <mat-card-actions style="padding:10px 25px;">
+            <button mat-raised-button color="primary" (click)="updatePostLikes()">LIKE</button>
+            <button mat-raised-button color="primary" (click)="text.focus()">COMMENT</button>
```

Comment:
uhhhh the comment button is not necessary i think... like i click on the text field it still provides the same purpose